tests/safeds/data/tabular/transformation/test_sequential_table_transformer.py

```diff
@@ -0,0 +1,215 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.data.tabular.transformation import (
+    Discretizer,
+    LabelEncoder,
+    OneHotEncoder,
+    SequentialTableTransformer,
+    SimpleImputer,
+    StandardScaler,
+    TableTransformer,
+)
+from safeds.exceptions import TransformerNotFittedError, TransformerNotInvertibleError
+
+from tests.helpers import assert_tables_equal
+
+
+class TestInit:
+
+    def test_should_raise_value_error_on_empty_list(self) -> None:
+        with pytest.raises(ValueError, match=("transformers must contain at least 1 transformer")):
+            SequentialTableTransformer(transformers=[])  # type: ignore  # noqa: PGH003
+
+
+class TestFit:
+    def test_should_raise_value_error_on_empty_table(self) -> None:
+        one_hot = OneHotEncoder()
+        imputer = SimpleImputer(SimpleImputer.Strategy.constant(0))
+        transformers = [one_hot, imputer]
```

Comment:
Tests should be as simple as possible. One transformer is enough to test that this error is raised.

Reply:
changed the test to only use a SimpleImputer